src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -560,12 +664,13 @@ public void ILLink_displays_informational_warning(string targetFramework)
         {
             var projectName = "HelloWorld";
             var referenceProjectName = "ClassLibForILLink";
+            var rid = EnvironmentInfo.GetCompatibleRid(targetFramework);
 
             var testProject = CreateTestProjectForILLinkTesting(targetFramework, projectName, referenceProjectName);
             var testAsset = _testAssetsManager.CreateTestProject(testProject);
 
             var publishCommand = new PublishCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name));
-            publishCommand.Execute("/p:PublishTrimmed=true", $"/p:SelfContained=true", "/p:PublishTrimmed=true")
+            publishCommand.Execute("/p:PublishTrimmed=true", $"/p:RuntimeIdentifier={rid}", "/p:SelfContained=true", "/p:PublishTrimmed=true")
```

Comment:
Why was this needed?

Reply:
When I added `IsExe` to the test project (it should have been set to begin with), this errored because no RID was specified. It worked when it was a library project where SelfContained doesn't do anything (and the test doesn't actually check anything about the produced files - just looks for a warning message being printed). The other tests pass a RID like they should.